skribl/word_list.txt

```diff
@@ -0,0 +1,13 @@
+zed
+jetson
+point cloud
+aruco tag
+navigation
+state machine
+perception
+percep-memes
```

Comment:
tbh percep memes aint looking so good these days

Reply:
should've added percep-rant if you want the real memes.
or maybe even dev-ops

Reply:
dev-ops can get pretty memey ngl, but we might want to keep it on the down low